data/json/recipes/food/dairy_products.json

```diff
@@ -14,6 +14,22 @@
     "//": "Book Things to do with milk.  Add curdled milk and cheese recipes to the book.  Also consider adding to brewing json Airag from this book.",
     "components": [ [ [ "milk_cream", 5 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "raw_butter",
+    "id_suffix": "shake",
+    "byproducts": [ [ "water", 1 ], [ "jar_glass_sealed", 1 ] ],
```

Comment:
```suggestion
    "byproducts": [ [ "buttermilk", 3 ], [ "jar_glass_sealed", 1 ] ],
```

Reply:
I guessed how much buttermilk you'd need to make the calorie count fall into the right average. It could be more or less.

Reply:
Ill test it now